roofit/roofitcore/inc/RooAbsOptTestStatistic.h

```diff
@@ -31,7 +31,7 @@ class RooAbsOptTestStatistic : public RooAbsTestStatistic {
   // Constructors, assignment etc
   RooAbsOptTestStatistic() ;
   RooAbsOptTestStatistic(const char *name, const char *title, RooAbsReal& real, RooAbsData& data,
-			 const RooArgSet& projDeps, const char* rangeName=0, const char* addCoefRangeName=0,
+			 const RooArgSet& projDeps, bool batchMode, const char* rangeName=0, const char* addCoefRangeName=0,
```

Comment:
In general, this will break interfaces for the people out there. If somebody derived their own class from this interface, though, you now broke their code.
*This one time here*, it's maybe OK, because I doubt that many people did actually write their own implementation, and call this constructor.

When you break an interface, the minimum is to add a comment in the release notes. If people advance to 6.26, and their code doesn't compile, they can at least find out that this was intended, and not a bug.

Reply:
It could be the last argument and have a default value so as not to break the interface, yes.